app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    count_of_non_vaccinated = 0
+    do_not_have_mask = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            count_of_non_vaccinated += 1
```

Comment:
You can immediately return the value in this exception "All friends should be vaccinated"